lib/model/autocomplete.dart

```diff
@@ -901,7 +900,7 @@ class MentionAutocompleteQuery extends ComposeAutocompleteQuery {
 
     return UserGroupMentionAutocompleteResult(
       groupId: userGroup.id,
-      rank: _rankUserGroupResult(userGroup, nameMatchQuality: nameMatchQuality));
+      rank: _rankUserGroupResult(nameMatchQuality));
```

Comment:
```
autocomplete [nfc]: Remove User(Group) params from _rankUser(Group)Result
```

Does some later work depend on this removal? The item being ranked seems like a natural thing for a "rank-item" method to want to examine as it does its work. For example, maybe for a refinement where we uprank user groups if the self-user is a member of the group.